<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="35" total="38" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="ODIN_API_CALL by groups" duration-ms="33303" started-at="2020-03-22T13:44:44Z" finished-at="2020-03-22T13:45:17Z">
    <groups>
      <group name="BG_MCC">
        <method signature="RequestProcessorTestMarius.test34()[pri:34, instance:ODIN_REQUEST_PROCESSOR_APITEST.RequestProcessorTestMarius@3a883ce7]" name="test34" class="ODIN_REQUEST_PROCESSOR_APITEST.RequestProcessorTestMarius"/>
        <method signature="RequestProcessorTestMarius.test35()[pri:35, instance:ODIN_REQUEST_PROCESSOR_APITEST.RequestProcessorTestMarius@3a883ce7]" name="test35" class="ODIN_REQUEST_PROCESSOR_APITEST.RequestProcessorTestMarius"/>
        <method signature="RequestProcessorTestMarius.test36()[pri:36, instance:ODIN_REQUEST_PROCESSOR_APITEST.RequestProcessorTestMarius@3a883ce7]" name="test36" class="ODIN_REQUEST_PROCESSOR_APITEST.RequestProcessorTestMarius"/>
      </group> <!-- BG_MCC -->
    </groups>
    <test name="GRP-BG_MCC" duration-ms="33303" started-at="2020-03-22T13:44:44Z" finished-at="2020-03-22T13:45:17Z">
      <class name="ODIN_REQUEST_PROCESSOR_APITEST.RequestProcessorTestMarius">
        <test-method status="PASS" signature="startTest()[pri:0, instance:ODIN_REQUEST_PROCESSOR_APITEST.RequestProcessorTestMarius@3a883ce7]" name="startTest" is-config="true" duration-ms="2579" started-at="2020-03-22T13:44:44Z" finished-at="2020-03-22T13:44:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startTest -->
        <test-method status="PASS" signature="test34()[pri:34, instance:ODIN_REQUEST_PROCESSOR_APITEST.RequestProcessorTestMarius@3a883ce7]" name="test34" duration-ms="9021" started-at="2020-03-22T13:44:46Z" finished-at="2020-03-22T13:44:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test34 -->
        <test-method status="PASS" signature="endTest()[pri:0, instance:ODIN_REQUEST_PROCESSOR_APITEST.RequestProcessorTestMarius@3a883ce7]" name="endTest" is-config="true" duration-ms="0" started-at="2020-03-22T13:44:55Z" finished-at="2020-03-22T13:44:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
        <test-method status="FAIL" signature="test35()[pri:35, instance:ODIN_REQUEST_PROCESSOR_APITEST.RequestProcessorTestMarius@3a883ce7]" name="test35" duration-ms="8149" started-at="2020-03-22T13:44:55Z" finished-at="2020-03-22T13:45:03Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Lists differ at element [1]: 2020-09-08 00:00:00 != 2020-10-10 00:00:00 expected [2020-09-08 00:00:00] but found [2020-10-10 00:00:00]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Lists differ at element [1]: 2020-09-08 00:00:00 != 2020-10-10 00:00:00 expected [2020-09-08 00:00:00] but found [2020-10-10 00:00:00]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:831)
at org.testng.Assert.assertEquals(Assert.java:796)
at ODIN_REQUEST_PROCESSOR_APITEST.RequestProcessorTestMarius.test35(RequestProcessorTestMarius.java:1247)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test35 -->
        <test-method status="PASS" signature="endTest()[pri:0, instance:ODIN_REQUEST_PROCESSOR_APITEST.RequestProcessorTestMarius@3a883ce7]" name="endTest" is-config="true" duration-ms="1" started-at="2020-03-22T13:45:03Z" finished-at="2020-03-22T13:45:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
        <test-method status="FAIL" signature="test36()[pri:36, instance:ODIN_REQUEST_PROCESSOR_APITEST.RequestProcessorTestMarius@3a883ce7]" name="test36" duration-ms="13300" started-at="2020-03-22T13:45:03Z" finished-at="2020-03-22T13:45:17Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[lists don't have the same size expected [2] but found [4]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: lists don't have the same size expected [2] but found [4]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:819)
at org.testng.Assert.assertEquals(Assert.java:796)
at ODIN_REQUEST_PROCESSOR_APITEST.RequestProcessorTestMarius.test36(RequestProcessorTestMarius.java:1331)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test36 -->
        <test-method status="PASS" signature="closeDB()[pri:0, instance:ODIN_REQUEST_PROCESSOR_APITEST.RequestProcessorTestMarius@3a883ce7]" name="closeDB" is-config="true" duration-ms="214" started-at="2020-03-22T13:45:17Z" finished-at="2020-03-22T13:45:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeDB -->
        <test-method status="PASS" signature="endTest()[pri:0, instance:ODIN_REQUEST_PROCESSOR_APITEST.RequestProcessorTestMarius@3a883ce7]" name="endTest" is-config="true" duration-ms="0" started-at="2020-03-22T13:45:17Z" finished-at="2020-03-22T13:45:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
      </class> <!-- ODIN_REQUEST_PROCESSOR_APITEST.RequestProcessorTestMarius -->
    </test> <!-- GRP-BG_MCC -->
  </suite> <!-- ODIN_API_CALL by groups -->
</testng-results>
